embeddings/pipeline/lightning_classification.py

```diff
@@ -35,10 +37,11 @@ def __init__(
         load_dataset_kwargs: Optional[Dict[str, Any]] = None,
         task_model_kwargs: Optional[Dict[str, Any]] = None,
         task_train_kwargs: Optional[Dict[str, Any]] = None,
+        predict_subset: Literal["dev", "test"] = "test",
```

Comment:
do we want to create an enum for sub-set names? `dev` / `val` / `validation` will not be mixed in future  :D

Reply:
Done